src/packages/searchbar/searchbar.taro.tsx

```
@@ -182,13 +190,16 @@
     onChange && onChange?.('')
     onClear && onClear(event)
   }
-  const onKeypress = (e: any) => {
-    if (e.key === 'Enter' || e.keyCode === 13) {
-      if (typeof e.cancelable !== 'boolean' || e.cancelable) {
-        e.preventDefault()
-      }
-      onSearch && onSearch(value as string)
-    }
+  //   const onKeypress = (event: any) => {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)